symposion/proposals/models.py

```diff
@@ -196,8 +196,9 @@ def speakers(self):
     def notification_email_context(self):
         return {
             "title": self.title,
-            "speaker": self.speaker.name,
+            "speaker": self.speaker,
             "speakers": ', '.join([x.name for x in self.speakers()]),
```

Comment:
Probably worth changing this variable name given that `speaker` now does something different